.github/workflows/daily_reports.yaml

```diff
@@ -124,6 +124,10 @@ jobs:
           restore-keys: |
             loc-report-
 
+      - name: Check previous loc report
+        run: |
+          cat loc_report.json
+
```

Comment:
I added this directive here to read the `loc_report.json` that was saved in the cache.